crescent/commands/decorators.py

```diff
@@ -60,28 +60,31 @@ class _Parameter(NamedTuple):
     default: Any
 
 
+def _unwrap_optional(origin: Type) -> Any:
```

Comment:
If it returns origin, shouldn't the return type also be Type?

Reply:
Line 68 returns `Any`